### ISSUE TITLE
Panic clicking playlist creator

### ISSUE THREAD

**Issue by juxuanu** [xou816/spot#433]
**Describe the bug**
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: http::Error(InvalidUri(InvalidUriChar))', src/api/client.rs:48:14
```

**To Reproduce**
1. Go to [this](https://open.spotify.com/playlist/4YOie8FbCkBPz36zUNGQx5?si=c85bb551f18a407f) playlist
2. Click on the creator username: ![imatge](https://user-images.githubusercontent.com/24759034/149560472-976fe0cf-90b8-40f4-ac4b-101ef3b19cbe.png)

**Expected behavior**
It should navigate to the user's profile, I guess with their playlists, etc.

**General information:**
 - Distribution: ArchLinux, Gnome
 - Installation method [e. g. built from source, installed from Flathub...]: Flathub
 - Version [e.g. 0.1.0]: 0.3.0
 - Device used [e. g. desktop, phone...]: desktop

**Stack trace:**
If applicable, run the application from a terminal and paste relevant log output.
```
➜ flatpak run --env=RUST_BACKTRACE=full dev.alextren.Spot
ATTENTION: default value of option mesa_glthread overridden by environment.
dri_create_context: requested glthread but driver is missing backgroundCallable V2 extension
dri_create_context: requested glthread but driver is missing backgroundCallable V2 extension
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: http::Error(InvalidUri(InvalidUriChar))', src/api/client.rs:48:14
stack backtrace:
   0:     0x55c0d511eeac - <unknown>
   1:     0x55c0d5143adc - <unknown>
   2:     0x55c0d51172f5 - <unknown>
   3:     0x55c0d5120ad0 - <unknown>
   4:     0x55c0d5120685 - <unknown>
   5:     0x55c0d5121184 - <unknown>
   6:     0x55c0d5120c60 - <unknown>
   7:     0x55c0d511f354 - <unknown>
   8:     0x55c0d5120bc9 - <unknown>
   9:     0x55c0d47b79d1 - <unknown>
  10:     0x55c0d47b7ac3 - <unknown>
  11:     0x55c0d47e0e12 - <unknown>
  12:     0x55c0d49df265 - <unknown>
  13:     0x55c0d49dc295 - <unknown>
  14:     0x55c0d4885efe - <unknown>
  15:     0x55c0d4935dd5 - <unknown>
  16:     0x55c0d49e90fd - <unknown>
  17:     0x55c0d49d63a4 - <unknown>
  18:     0x55c0d49b814f - <unknown>
  19:     0x55c0d49efadd - <unknown>
  20:     0x55c0d4a1a96e - <unknown>
  21:     0x55c0d50fa027 - <unknown>
  22:     0x55c0d50f5cca - <unknown>
  23:     0x7f0f5bda76d1 - g_main_context_dispatch
  24:     0x7f0f5bda7bb8 - <unknown>
  25:     0x7f0f5bda7c83 - g_main_context_iteration
  26:     0x7f0f5bfdac45 - g_application_run
  27:     0x55c0d48bad7e - <unknown>
  28:     0x55c0d493fcb3 - <unknown>
  29:     0x7f0f5bda8a15 - g_main_context_invoke_full
  30:     0x55c0d4882fb1 - <unknown>
  31:     0x55c0d486cd03 - <unknown>
  32:     0x55c0d4961dd9 - <unknown>
  33:     0x55c0d511eb8b - <unknown>
  34:     0x55c0d4883632 - main
  35:     0x7f0f5ba0db90 - __libc_start_main
  36:     0x55c0d47b81fe - <unknown>
  37:                0x0 - <unknown>
```

**Comment by xou816** [comment]
Hi! Thanks for the report. Does it happen with every playlist? I can´t reproduce it with mine. If there's a public playlist you'd be able to share, that'dbe great!

**Comment by juxuanu** [comment]
Hi! You can check the link in the step 1 for the playlist that makes this happen. 😊

**Comment by xou816** [comment]
My bad, missed it, thanks! I'll have a look!

**Comment by xou816** [comment]
Should be fixed in 0.3.1!

**Comment by juxuanu** [comment]
It is, and I can see that username now has its own test, haha!
I can't the see heart in Spot, though! I don't know if that's expected. 
![imatge](https://user-images.githubusercontent.com/24759034/150792187-0719c074-5ac0-408c-9686-956c0f3522c3.png)
